pinot-common/src/main/java/org/apache/pinot/common/utils/time/TimeUtils.java

```diff
@@ -108,7 +111,16 @@ public static TimeUnit timeUnitFromString(@Nullable String timeUnitString) {
    * <p>The current valid range used is between beginning of 1971 and beginning of 2071.
    */
   public static boolean timeValueInValidRange(long timeValueInMillis) {
-    return (VALID_MIN_TIME_MILLIS <= timeValueInMillis && timeValueInMillis <= VALID_MAX_TIME_MILLIS);
+    return timeValueInMillis >= VALID_MIN_TIME_MILLIS && timeValueInMillis <= VALID_MAX_TIME_MILLIS;
```

Comment:
Not sure why this change was made

Reply:
Just want keep the `timeValueInValidRange()` and `isValidTimeInterval()` checking order consistent.